.coveragerc

```diff
@@ -1,20 +0,0 @@
-[run]
-source = zope.testrunner
-parallel = true
-omit =
-    # These files are definitely imported and executed,
-    # but for some reason don't show up as such.
-    # See testrunner-knit
-    src/zope/testrunner/tests/testrunner-ex-pp-products/more/sampletests.py
-    src/zope/testrunner/tests/testrunner-ex-pp-products/sampletests.py
```

Comment:
I suppose these need to be moved into `meta.toml` and regenerated into wherever that template puts coverage configs